pkg/network/setup/network_test.go

```diff
@@ -288,6 +300,79 @@ var _ = Describe("VMNetworkConfigurator", func() {
 			})
 		})
 	})
+
+	Context("pod network phase1", func() {
```

Comment:
It seems that the tested VM has pod network only, what about secondary networks? 
Dont we need to test VMI with secondary networks as well?

Reply:
These tests cover the code in `network.go`, not the logic in `configstate.go`, for that there are closer tests.
And the only scenario these tests focus on is the correct error returning (i.e. regular or critical).